cmd/main.go

```diff
@@ -190,9 +194,29 @@ func setupProxmoxClient(ctx context.Context, logger logr.Logger) (capmox.Client,
 	if ProxmoxURL == "" || ProxmoxTokenID == "" || ProxmoxSecret == "" {
 		return nil, nil
 	}
-	// TODO, check if we need to delete tls config
+
+	rootCerts, err := x509.SystemCertPool()
```

Comment:
Yipp, if the container CAPMOX is runnig in does not provide a system store this will error. Need to have a look whether I can differentiate that from other loading errors… 🤔